homeassistant/components/keenetic_ndms2/router.py

```diff
@@ -117,6 +125,18 @@ def signal_update(self):
         """Event specific per router entry to signal updates."""
         return f"keenetic-update-{self.config_entry.entry_id}"
 
+    async def set_interface_state(self, interface_id: str, is_on: bool):
+        self._client.set_interface_state(interface_id, is_on)
+
+        async def deferred_update(_now):
+            await self.request_update()
+
+        async_call_later(
```

Comment:
I think this should not accumulate. 

`async_call_later` returns a cancelling callback. Need to store it so when new `set_interface_state` request comes we cancel any pending `deferred_update`